src/features/unified-share-modal/utils/convertData.js

```diff
@@ -458,30 +458,45 @@ export const convertCollabsResponse = (
  */
 export const convertCollabsRequest = (
     collabRequest: InviteCollaboratorsRequest,
+    collaboratorsList: collaboratorsListType | null,
```

Comment:
IMO this and `collabSet` below could benefit from being renamed to something like "existingCollaboratorsList" to help distinguish it from the collabs being added in the request